src/dune/coq_lib_name.ml

```diff
@@ -1,29 +1,32 @@
 (* This file is licensed under The MIT License *)
 (* (c) MINES ParisTech 2018-2019               *)
 (* (c) INRIA 2020                              *)
-(*     Written by: Emilio Jesús Gallego Arias  *)
+(* Written by: Emilio Jesús Gallego Arias *)
```

Comment:
This seems suspicious

Reply:
I get the same diff with OCamlformat 0.12.
This is caused by the `wrap-comments=true` option in `.ocamlformat`. Removing it doesn't add more diffs.

Reply:
Ok thanks, I guess I'll adapt the end-of-comment wrapping then.